tests/unit/__init__.py

```diff
@@ -771,7 +790,7 @@ def setUp(self):
         }
 
         self.account_service_url = '/start/'
-        self.account_service_log_out_url = '/start/save-and-exit/'
+        self.account_service_log_out_url = '/signed-out/'
```

Comment:
account_service_log_out_url in eq_payload json, line 782 hasn't been updated as per this new log out url.